test/box-luatest/gh_6452_mvcc_crash_in_prepare_test.lua

```diff
@@ -0,0 +1,84 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+g.before_all = function()
+    g.server = server:new{
+        alias   = 'default',
+        box_cfg = {memtx_use_mvcc_engine = true}
+    }
+    g.server:start()
+end
+
+g.after_all = function()
+    g.server:drop()
+end
+
+g.test_mvcc_crash_in_prepare_case1 = function()
+    g.server:exec(function()
+        local t = require('luatest')
+        local txn_proxy = require("test.box.lua.txn_proxy")
+
+        box.cfg{memtx_use_mvcc_engine = true}
+        local s = box.schema.space.create("s", {engine="memtx"})
+        s:create_index("pk", {type="tree"})
+        s:create_index("sk", {parts={2}, type="hash"})
```

Comment:
Please consider factoring this code out into `g.before_each`, e.g.:
https://github.com/tarantool/tarantool/blob/f1d750f036956873c79bf5de51f77b38da25ae0f/test/vinyl-luatest/update_optimize_test.lua#L25-L33